server.js

```diff
@@ -0,0 +1,7 @@
+import app from './app';
+
+app.set('port', process.env.PORT || 3001);
+
+app.listen(app.get('port'), () => {
+  console.log(`${app.locals.title} is running on localhost:${app.get('port')}.`);
+});
```

Comment:
Nice great job separating this from the app.js file. Ran into issues with testing related to the port and listening. Guess that is something that isn't mocked and best just to omit from the testing environment